src/presets.js

```diff
@@ -169,6 +196,32 @@ const attachNestedChangelog = (upstreams, commit, callback) => {
 					cb,
 				);
 			}
+			const regexp1 = new RegExp(`[Uu]pdate ${upstream.pattern}`);
```

Comment:
Let's rename `regexp1` and `match1` to `regexp` and `match`.

Reply:
@stathismor we already have `regexp` and `match` const above in this function so they cannot be redefined unless we don't use consts. Any other variable name suggestion you prefer?